test-utils.ts

```diff
@@ -14,9 +14,12 @@ export const loadTwigTemplate = (templatePath: string) => {
   const templatePromise = twing.load(
     path.relative(process.cwd(), templatePath)
   );
-  // Using "await" here because next version of twing returns promises
-  // eslint-disable-next-line @cloudfour/typescript-eslint/await-thenable
-  return async (data: any = {}) => (await templatePromise).render(data);
```

Comment:
eslint-plugin-unicorn doesn't want you to do `(await something).something` anymore. I don't love the rule but I don't hate it enough to disable it. The rule advocates for separating the awaited object into its own variable before retrieving variables from it. I am mostly indifferent